composer.json

```diff
@@ -26,7 +26,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
+        "php": "~7.2.22 || ^7.3.9",
```

Comment:
Im also not super happy about changing the PHP requirements. It is set to 7.2.5 because of that is the lowest, most supported PHP version on major Linux LTSes.

Reply:
IMO, regardless how available is a

Reply:
Are you saying the Guzzle is unusable when running 7.2.5?

Reply:
I'm not saying it is unusable, just that a feature provided by a dependency (the CURL extension) is not behaving as expected.
I noticed the issue since a piece of code in a private project was checking the error message (an edge case, like you mentioned).